src/Restorer/AbstractRestorer.php

```diff
@@ -93,6 +97,12 @@ public function isVerbose(): bool
         return $this->verbose;
     }
 
+    #[\Override]
+    protected function beforeProcess(): void
+    {
+        $this->process->setTimeout(1800);
```

Comment:
I'm still concerned about this timeout, it should be configurable.

Reply:
I agree, but this is the default timeout. Make the timeout configurable is another subject.

Reply:
https://github.com/makinacorpus/DbToolsBundle/issues/102